libraries/Microsoft.Bot.Builder.Classic/DisableDelaySign.cmd

```diff
@@ -0,0 +1,5 @@
+@rep -find:"[assembly: AssemblyKeyFileAttribute" -replace:"//[assembly: AssemblyKeyFileAttribute" -r AssemblyInfo.cs
```

Comment:
We don't do this anywhere in our current build, that I know of. 

Can this be removed?